requirements.sh

```diff
@@ -1,17 +1,22 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Better to take command from environment, rather than hardcode path to executable.